ci/scripts/build-project.sh

```diff
@@ -4,14 +4,18 @@ set -euo pipefail
 
 readonly ONLY_SHOW_STANDARD_STREAMS_ON_TEST_FAILURE="${ONLY_SHOW_STANDARD_STREAMS_ON_TEST_FAILURE:-true}"
 readonly SKIP_TESTS="${SKIP_TESTS:-false}"
-readonly DISTRIBUTION_REPOSITORY_OUTPUT="${DISTRIBUTION_REPOSITORY_OUTPUT:?must be set}"
 
-if [ "$SKIP_TESTS" == "true" ]; then
+# shellcheck source=common.sh
+source "$(dirname "$0")/common.sh"
+repository=$(pwd)/distribution-repository
+#if [ "$SKIP_TESTS" == "true" ]; then
```

Comment:
Is this intentional?

Reply:
ah, definitely not. I was testing it locally so wanted it to run faster. Just force-pushed it. Thanks!